Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:18.16.0 as builder
+WORKDIR /src
+COPY ./andromeda-ui/package.json ./
+RUN npm install
+COPY ./andromeda-ui/. .
+RUN npm run build
+
+FROM python:3.11
+
+COPY ./andromeda/requirements.txt /tmp/requirements.txt
+WORKDIR /tmp
+RUN pip3 install gunicorn && pip3 install -r requirements.txt
+
+COPY ./andromeda/. /api
+COPY ./app.py /api/app.py
```

Comment:
Could we move app.py to the `andromeda` directory with the rest of the python code? 
Then we could delete this line.